shared/js/background/tracker-utils.js

```diff
@@ -167,8 +172,8 @@ function truncateReferrer (referrer, target) {
         return undefined
     }
 
-    if (tdsStorage.ReferrerExcludeList && tdsStorage.ReferrerExcludeList.excludedReferrers) {
-        const excludedDomains = tdsStorage.ReferrerExcludeList.excludedReferrers.map(e => e.domain)
+    if (tdsStorage.config.features.referrer && tdsStorage.config.features.referrer.exceptions) {
+        const excludedDomains = tdsStorage.config.features.referrer.exceptions.map(e => e.domain)
```

Comment:
it looks like we are doing exceptions differently here than everywhere else? we are looking for exact match instead of doing wildcard matching e.g. if "test.example.com" will be on the exception list, this exception will never match because we are doing `tldts.parse(referrer).domain` wich will return "example.com". Seems like a bug.

Should we use `isBrokenList` here instead?